src/recovery/hystart.rs

```diff
@@ -80,6 +88,26 @@ impl Hystart {
         Self {
             enabled,
 
+            last_round_min_rtt: Duration::MAX,
+
+            current_round_min_rtt: Duration::MAX,
+
+            css_baseline_min_rtt: Duration::MAX,
+
+            ..Default::default()
+        }
+    }
+
+    pub fn reset(&mut self) {
+        *self = Self {
```

Comment:
Can probably just do:
```rust
*self = Self::new(self.enabled);
```
to avoid duplicating.

Reply:
thanks. fixed